bedrock/firefox/views.py

```diff
@@ -507,6 +507,12 @@ def new(request):
                 template = 'firefox/new/break-free/scene1.html'
             elif experience == 'wayofthefox':
                 template = 'firefox/new/way-of-the-fox/scene1.html'
+            elif experience == 'privatenotoption':
+                template = 'firefox/new/private-not-option/scene1.html'
+            elif experience == 'conformitynotdefault':
+                template = 'firefox/new/conformity-not-default/scene1.html'
+            elif experience == 'browseuptoyou':
+                template = 'firefox/new/browse-up-to-you/scene1.html'
```

Comment:
Making sure it's okay that all these new templates link to the same view/UX of scene 2.